### PR TITLE
Client login setup

### PR DESCRIPTION
**Pull request by tristan-swu**
Might need to add this to /etc/hosts

```
127.0.0.1      battlesnake.local api.battlesnake.local
```

Msg me if you want some GH tokesn for env-local.yml

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -15,8 +16,9 @@
   },
   "scripts": {
     "build-css": "node-sass-chokidar src/ -o src/",
-    "watch-css": "npm run build-css && node-sass-chokidar src/ -o src/ --watch --recursive",
-    "start-js": "react-scripts start",
+    "watch-css":
+      "npm run build-css && node-sass-chokidar src/ -o src/ --watch --recursive",
+    "start-js": "HOST=battlesnake.local react-scripts start",
```
Comment (daniel-swu) [inline client/package.json]: why the env var here? this will probably require a /etc/hosts entry right?

File: client/src/App.js [hunk 2]
```diff
@@ -1,26 +1,70 @@
...
+    state = {
+      loading: true,
+      loggedIn: null,
+      error: null,
+      user: {}
+    };
+
+    componentDidMount = async () => {
+      try {
+        const user = await axios.get(`${config.SERVER}/self`, {
```
Comment (daniel-swu) [inline client/src/App.js]: axios better then fetch?

File: client/package.json (hunk 1 above)
Comment (tristan-swu) [reply client/package.json]: I can take this out - was left in on accident

File: server/src/app.ts [hunk 3]
```diff
@@ -42,29 +44,18 @@ app.use('/self', userRouter)
 app.use('/team', teamRouter)
 app.use('/tournaments', tournyRouter)
 
-nunjucks.configure('views', {
```
Comment (daniel-swu) [inline server/src/app.ts]: you delete all my stuff 😢

File: client/src/App.js (hunk 2 above)
Comment (tristan-swu) [reply client/src/App.js]: Rob already hooked it up so I just went with it

File: client/package.json (hunk 1 above)
Comment (daniel-swu) [reply client/package.json]: guessing it's for cookies

**Review by daniel-swu — APPROVED** [review]
feel free to merge whenever

**Pull request merged**